### DISCUSSION TITLE
Saving presets?

### DISCUSSION THREAD

**Discussion by Hugo-Matias** [LostRuins/koboldcpp#127] [Q&A] [answered]
Is there a way to save presets for settings, scenerios, etc? I'm fine doing it manually in system files but I couldn't find any way for doing this.

**Comment by LostRuins** [comment, marked as answer]
No, presets are fixed - but your custom settings can be saved into the .json save files by enabling `Export Settings` in options.

**Reply by Hugo-Matias** [reply]
Ah I see, thanks for the fast reply, cheers!
